code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm

```diff
@@ -102,7 +102,7 @@
 	loot = list()
 	stat_attack = HARD_CRIT
 	robust_searching = 1
-	food_type = list(/obj/item/reagent_containers/food/snacks/customizable/salad/ashsalad, /obj/item/reagent_containers/food/snacks/customizable/soup/ashsoup, /obj/item/food/grown/ash_flora)//use lavaland plants to feed the lavaland monster
+	food_type = list(/obj/item/food/grown/ash_flora)//use lavaland plants to feed the lavaland monster
```

Comment:
What happened to the other 2?

Reply:
custom food was killed for the refractor so they are not really possible to make atm. (this was sanctioned by the king)

Reply:
maybe some day